recipe/meta.yaml

```diff
@@ -1,14 +1,15 @@
+{% set python_min = '3.7' %}
```

Comment:
Technically the lowest version of python that the conda-forge ecosystem supports is `3.9`. You may run into issues if you pin the min lower than that.